### PR TITLE
WTHM second lecture

### PR DESCRIPTION
**Pull request by LostInDarkMath**

### CODE REVIEW

File: WTHM/chapter1.tex [hunk 1]
```diff
@@ -91,6 +91,267 @@ \section{Bedingtet Erwartungswert als $L_2$-Projektion}
...
+\begin{align*}
+X_n:=\left\lbrace\begin{array}{cl}
+X, & \falls |X|\leq n\\
+n, & \falls X>n\\
+-n, & \falls X<-n
+\end{array}\right.
+\end{align*}
+Mit dominanter Konvergenz gilt 
+\begin{align*}
+\limn\E[X_n-X|]=\E\left[\limn|X_n-X|\right]=0.
```
Comment (RobWalt) [inline WTHM/chapter1.tex]: ```|``` is missing

File: WTHM/chapter1.tex [hunk 2]
```diff
@@ -91,6 +91,267 @@ \section{Bedingtet Erwartungswert als $L_2$-Projektion}
...
+\end{align*}
+Mit Kontraktionseigenschaft gilt:
+\begin{align*}
+&\E\Big[\big|\E[X_m~|~\F]-\E[X_n~|~\F]\big|\Big]
+\leq\E\Big[\E\big[|X-M-X_n|~|~\big|~\F\Big]
```
Comment (RobWalt) [inline WTHM/chapter1.tex]: Something went wrong here with the ```|``` signs

File: WTHM/chapter1.tex [hunk 3]
```diff
@@ -91,6 +91,267 @@ \section{Bedingtet Erwartungswert als $L_2$-Projektion}
...
+\E[X\cdot\indi_{F^+}]-\E[Y\cdot\indi_{F^+}]\\
+&\stackeq{\text{(b)}}
+0\\
+&\Longrightarrow\P(F^+)=0
+\end{align*}
+Analog erhält man $\P(F^-)=0$. Also folgt insgesamt $\E[X~|~\F]=Y$ fast sicher.
+\end{proof}
+
+\begin{bemerkung}
+Die ``unbedingte Erwartung'' $\E[X]$ können wir als Spezialfall der bedingten Erwratung $\E[X~|~\F]$ für die triviale $\sigma$-Algebra $\F:=\lbrace\emptyset,\Omega\rbrace\subseteq\A$ auffassen.\\
```
Comment (RobWalt) [inline WTHM/chapter1.tex]: Typo

File: WTHM/chapter1.tex [hunk 4]
```diff
@@ -91,6 +91,267 @@ \section{Bedingtet Erwartungswert als $L_2$-Projektion}
...
+Denn es gilt:
+\begin{align*}
+\E[X\cdot\indi_\Omega]&=\E[X]=\E\big[\E[X]\cdot\indi_\Omega\big]\\
+\E[X\cdot\indi_\Omega]&=0=\E\big[\E[X]\cdot\indi_\emptyset\big]\\
```
Comment (RobWalt) [inline WTHM/chapter1.tex]: ```\Omega``` should be ```\emptyset```

File: WTHM/chapter1.tex [hunk 5]
```diff
@@ -91,6 +91,267 @@ \section{Bedingtet Erwartungswert als $L_2$-Projektion}
...
+Sei $X:\Omega\to\R$ eine Zufallsvariable auf $(\Omega,\A,\P)$ und $\F\subseteq\A$ Unter-$\sigma$-Algebra von $\A$. Dann heißt $X$ \textbf{unabhängig} von $\F$, in Zeichen $X\unab\F$
+\begin{align}\label{sigmaAlgebraUnabhaengig}
+:\Longleftrightarrow
+\E[f(X)\cdot\indi_A]
+=
+\E[f(X)]\cdot\P(A)\qquad\forall A\in\F\text{ und $f:\R\to\R$ beschränkt, Borel-messbar}
+\end{align}
+$X$ heißt \textbf{unabhängig} von einer Zufallsvariablen $Y$, in Zeichen $X\unab Y$ wenn eine der folgenden äquivalenten Bedingungen gilt:
+\begin{enumerate}[label=(\alph*)]
+\item $X\amalg\sigma(Y)$
```
Comment (RobWalt) [inline WTHM/chapter1.tex]: ```\amalg``` should be ```\unab```

File: WTHM/chapter1.tex [hunk 6]
```diff
@@ -91,6 +91,267 @@ \section{Bedingtet Erwartungswert als $L_2$-Projektion}
...
+\E[X\cdot\indi_\Omega]&=\E[X]=\E\big[\E[X]\cdot\indi_\Omega\big]\\
+\E[X\cdot\indi_\Omega]&=0=\E\big[\E[X]\cdot\indi_\emptyset\big]\\
+&\stackrel{\text{Thm 1.8}}{\Longrightarrow}
+\E[X]=\E[X~|~\F]\text{, da $\F$ trivial.}
+\end{align*}
+\end{bemerkung}
+
+\section*{Bedingter Erwartungungswert + Unabhängigkeit} %keine Nummer
+\begin{defi}
+Sei $X:\Omega\to\R$ eine Zufallsvariable auf $(\Omega,\A,\P)$ und $\F\subseteq\A$ Unter-$\sigma$-Algebra von $\A$. Dann heißt $X$ \textbf{unabhängig} von $\F$, in Zeichen $X\unab\F$
```
Comment (RobWalt) [inline WTHM/chapter1.tex]: make ```\unab``` prettier

Comment (LostInDarkMath) [reply WTHM/chapter1.tex]: Das wäre echt wünschenswert! Vielleicht hat @phijor eine Idee?

**Comment by LostInDarkMath** [comment]
Test

**Pull request merged**